docs/source/commandline/commandline-all.rst

```diff
@@ -500,16 +1034,41 @@ Core Search Path Options
 
     The default is ``.`` (that is, in the current working directory).
 
+    Example:
+        .. code-block::
+
+            mame64 -homepath c:\mame\lua
+
 .. _mame-commandline-rompath:
 
-**-rompath** / **-rp** / **-biospath** / **-bp** *<path>*
+**-rompath** / **-rp** *<path>*
 
     Specifies one or more paths within which to find ROM or disk images.
     Multiple paths can be specified by separating them with semicolons.
 
     The default is ``roms`` (that is, a directory ``roms`` in the current
     working directory).
 
+    Example:
+        .. code-block::
+
+            mame64 -rompath c:\mame\roms;c:\roms\another
+
+.. _mame-commandline-biospath:
+
+**-biospath** / **-bp** *<path>*
```

Comment:
Does this actually exist?  I don’t see it in `-showusage` and MAME searches for BIOS sets in the same ROM paths it uses for everything else.

Reply:
I checked the code – this is just an alias for `rompath`.  Either mention it as an alias for `rompath` or don’t mention it at all.  It’s probably something to deprecate as more likely to cause confusion than actually help.